### ISSUE TITLE
conflicting versions of ipywidgets and widgetsnbextension in conda

### ISSUE THREAD

**Issue by scottdraves** [jupyter-widgets/ipywidgets#1678]
if i run the following:
```
conda create -y -n testw python=3.5 jupyter pandas
source activate testw
conda list | grep widgets
```
I get 
```
ipywidgets                6.0.0                    py35_0
widgetsnbextension        3.0.1                    py35_0
```
and indeed when the client tries to instantiate a widget, it fails as follows:
<img width="706" alt="screen shot 2017-08-27 at 11 26 16 pm" src="https://user-images.githubusercontent.com/963093/29758144-2d41ae76-8b7f-11e7-803f-17d9ac6d4ce4.png">
This can be fixed with
```
conda install widgetsnbextension=2.0.0
```
Note I tried removing pandas and using python 3.6 and it made no difference in what conda packages were installed but i did not actually try running jupyter to test further.
note: i am testing on a mac.

**Comment by SylvainCorlay** [comment]
@scottdraves This would be an issue with the conda-forge conda recipe.

In both [ipywidgets 7.0](https://github.com/conda-forge/ipywidgets-feedstock/commit/83408c22b722a865093dd08e99300c3e34a09ea6) and [ipywidgets 6.0](https://github.com/conda-forge/ipywidgets-feedstock/commit/beb87959189ccd0e67dbadba5a960690aaed8438) we use `>=` to set the dependency to widgetsnbextension. We should have added an upper bound in both case corresponding to the next major version.

**Comment by jasongrout** [comment]
I thought I opened an issue for this. We need to release a new build of 6.0 and release 6.0.1 with some fixes, both with updated dependencies. That's on my list to do for today.

**Comment by jasongrout** [comment]
Oh, I think I remember - I had posted to the conda-forge about how to release previous patch releases.

Basically, we'll need to create a 6.x branch in conda-forge, and merge a new build of 6.0 with the metadata fix, and then release a 6.0.1 with a few fixes backported

**Comment by jasongrout** [comment]
We released a 6.0 build number 1 conda-forge build which should fix the versioning problem, and we're currently releasing a 6.0.1 with a few fixes.
